js/main.js

```diff
@@ -119,3 +140,21 @@ document.addEventListener('DOMContentLoaded', function () {
     viewSwap('profile');
   }
 });
+
+document.addEventListener('click', function (event) {
+  if (event.target.tagName !== 'A') {
+    return;
+  }
+  if (event.target.tagName === 'A') {
```

Comment:
You can remove this conditional, you want the user to be able to go to `profile` if they have user data, and you are already redirecting to `edit-profile` if they don't in the conditional below 👍